### ISSUE TITLE
 The file "Foo.swift" couldn’t be opened with absolute paths

### ISSUE THREAD

**Issue by keith** [yonaskolb/XcodeGen#88]
With this config:

```yaml
name: Project
targets:
  Project:
    type: application
    platform: iOS
    sources:
      - Sources/foo.swift
```

And this file structure:

```
.
├── Sources
│   └── Foo.swift
└── project.yml

1 directory, 2 files
```

`xcodegen` fails with this error message:

```
📋  Loaded spec:
  Name: Project
  Targets:
    📱  application: Project
Generation failed: The file “Foo.swift” couldn’t be opened.
```

Should you be able to reference a path like this absolutely? This is useful for when you have more complex file structures where sources and tests are nearby, meaning you can't reference the entire directory or you'll include test files in the non-test target.

Sample project: [XcodeGenTest2.zip](https://github.com/yonaskolb/XcodeGen/files/1389018/XcodeGenTest2.zip)

**Comment by yonaskolb** [comment]
Yeah I guess this should be made possible, as being able to `exclude` files from a source directory hasn't landed yet either

**Comment by yonaskolb** [comment]
This was resolved in #106
